styles/themes/teammember/_login-header.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../identity';
+@import 'variables';
+
+$login-area-padding-top: 43px;
+$login-logo-margin-right: 24px;
+$logo-separator-height: 41px;
+$logo-separator-width: 1px;
+$workwise-logo-margin-left: 24px;
+$workwise-logo-margin-top: 6px;
+$login-heading-padding-bottom: 16px;
+$login-heading-width: 282px;
+$logo-separator-background-color: rgba(243, 216, 162, 0);
+$logo-separator-box-shadow-color: rgba(23, 54, 101, 0.5);
+
+.login-area {
+    clear: both;
+    margin: 0 auto;
+    padding-top: $login-area-padding-top;
+    text-align: center;
+    width: $login-area-v2-width;
+
+    a {
+        color: $login-font-color;
+        text-decoration: none;
+    }
+
+    .login-logo {
+        float: left;
+        margin-right: $login-logo-margin-right;
+    }
+
+    .logo-separator {
+        background-image: linear-gradient(-180deg, $logo-separator-background-color 0%, $color-secondary-light 51%, $logo-separator-background-color 97%);
+        box-shadow: 1px 0 2px 0 $logo-separator-box-shadow-color;
+        float: left;
+        height: $logo-separator-height;
+        width: $logo-separator-width;
+    }
+
+    .work-wise {
+        color: $color-base-20;
```

Comment:
I would consider calling this 'application-name', or 'product-name', etc. When the css is moved out of /themes it will no longer be specific to workwise.